src/main/java/net/mvndicraft/townywaypoints/Waypoint.java

```diff
@@ -50,6 +52,10 @@ public boolean isSea()
     {
         return sea;
     }
+    public boolean isTravelWithHorses()
```

Comment:
minor nitpick but could this be travelWithVehicle?

Reply:
Yes, I forgot to change it there